kubernetes-ingress/tls-with-cert-manager.md

```diff
@@ -0,0 +1,121 @@
+# Ingress example with TLS termination using cert-manager
+
+This example builds on the [HTTP](http.md) and [gRPC](grpc.md) ingress
+examples, adding TLS termination. This example is similar to the
+[TLS](tls.md) ingress example, in which you had to manually create a
+self-signed CA using `minica`. In this example, you will be using
+[cert-manager](https://cert-manager.io/), a Kubernetes-native operator that
+issues and renews TLS certificates for you.
+
+## Create TLS certificate and private key
+
+Let us install cert-manager:
+
+```sh
+helm repo add jetstack https://charts.jetstack.io
+helm install cert-manager jetstack/cert-manager --version v1.7.1 --namespace cert-manager --set installCRDs=true --create-namespace
+```
+
+Now, create a CA Issuer:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/cilium/cilium-service-mesh-beta/main/kubernetes-ingress/ca-issuer.yaml
+```
+
+## Deploy the ingress
+
+If you previously ran the HTTP and/or gRPC ingress demos, delete the ingresses:
+
+```sh
+kubectl delete ingress basic-ingress grpc-ingress
+```
+
+The ingress configuration for this demo provides the same routing as those demos
+but with the addition of TLS termination. Deploy this ingress:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/cilium/cilium-service-mesh-beta/main/kubernetes-ingress/tls-ingress.yaml
+```
```

Comment:
I think we could avoid duplicating a lot of the material from the existing example

```suggestion
## Deploy the ingress

Follow the instructions from the [TLS demo](TLS.md) to deploy the ingress. 
```

Reply:
Good idea. I fixed this 06045ec210bbea3246355602364105b6df3e3e0a. Thanks!